src/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "AHB Tabellen - Anwendungshandbücher für Menschen",
+  "short_name": "AHB Tabellen",
+  "icons": [
+    {
+      "src": "/assets/favicon/hf_logo_192x192.png.png",
+      "sizes": "192x192",
+      "type": "image/png"
+    },
+    {
+      "src": "/assets/favicon/hf_logo_512x512.png",
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ],
+  "theme_color": "#EBBEC1",
+  "background_color": "#EBBEC1",
+  "display": "standalone"
+}
```

Comment:
We should leave this out. If you want pwa functionality we should follow this guide: https://angular.io/guide/service-worker-getting-started#adding-a-service-worker-to-your-project

Reply:
removed in c5567d1f663ccfb759bfc35a77e6a573e7bb8a5f